From faa1600fa5f0b6d2b15b23e8fc602be4eeea2580 Mon Sep 17 00:00:00 2001
From: sveinung <sveinung@a0f10bec-cc02-0410-94fc-a9cfff90b4cd>
Date: Fri, 28 Aug 2015 10:54:14 +0000
Subject: [PATCH 1/4] Make 7.15.4 the minimum CURL version

and document what CURL version is required in INSTALL

See bug #23824

git-svn-id: svn://svn.gna.org/svn/freeciv/trunk@29702 a0f10bec-cc02-0410-94fc-a9cfff90b4cd
---
 INSTALL              | 2 +-
 configure.ac         | 2 +-
 doc/README.packaging | 1 +
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/INSTALL b/INSTALL
index ac88b41..6375798 100644
--- a/INSTALL
+++ b/INSTALL
@@ -73,7 +73,7 @@ out whether your system is suitable.  If in doubt, just try it.
 
    The output should include "GNU Make" somewhere.
 
- - libcurl
+ - libcurl version 7.15.4 or better
 
    Libcurl takes care of http communication with the metaserver, and
    file transfers of modpack installers
diff --git a/configure.ac b/configure.ac
index bf85fa7..ff43452 100644
--- a/configure.ac
+++ b/configure.ac
@@ -543,7 +543,7 @@ PKG_PROG_PKG_CONFIG
 if test "x$PKG_CONFIG" = "x" ; then
   AC_MSG_ERROR([pkg-config not found])
 fi
-PKG_CHECK_MODULES([CURL], [libcurl >= 7.9.7],,
+PKG_CHECK_MODULES([CURL], [libcurl >= 7.15.4],,
     [AC_MSG_ERROR([libcurl development files required])])
 UTILITY_CFLAGS="${UTILITY_CFLAGS} ${CURL_CFLAGS}"
 UTILITY_LIBS="${UTILITY_LIBS} ${CURL_LIBS}"
diff --git a/doc/README.packaging b/doc/README.packaging
index c0a6f9a..464cdbc 100644
--- a/doc/README.packaging
+++ b/doc/README.packaging
@@ -26,6 +26,7 @@ Updating from 2.6 to 3.0
   themselves. This has obsoleted old --with-modlist configure option.
   See chapter "Project definition file" below.
 * Xaw-client has been dropped completely
+* Minimum libcurl version is now 7.15.4
 
 ----------------------------------------------------------------------
 Compatibility of modified versions
-- 
2.1.4

